packages/silk-datastore-syncer/spec

```diff
@@ -0,0 +1,8 @@
+---
+name: silk-datastore-syncer
+
+dependencies:
+  - golang-1.19-linux
+
+files:
+  - code.cloudfoundry.org/**/*
```

Comment:
The rest of the silk packages are more specific in identifying which files are necessary, maybe we should be too?

Reply:
Yeah we should stay consistent with the rest of the release. We'll add it to the sync-package specs script + get this fixed after merging. thanks for calling this out!